### ISSUE TITLE
Optimize pointer event handling on Web

### ISSUE THREAD

**Issue by dkwingsmt** [flutter/flutter#134172]
## Details

**Target Platform:**
**Target OS version/browser:**
**Devices:**

## Logs

<details>
<summary>Logs</summary>

```
```

```
```

</details>

**Comment by yjbanov** [comment]
@dkwingsmt Were you able to find any actionable information since you filed the issue?

**Comment by dkwingsmt** [comment]
No, I haven't worked on it yet. I don't expect to work on it for a while, since I'm currently fully on multi-view.

**Comment by dkwingsmt** [comment]
It's been a while, but I think the last time I profiled it, I found that the symbols of the Web profile during a frame was highly flaky. Very often a tracing will be lost (or be recorded out of order, maybe) so that multiple events are combined into one, creating the illusion that it's the pointer event handling that took a large portion of time.
